packages/@internationalized/date/docs/ZonedDateTime.mdx

```diff
@@ -0,0 +1,486 @@
+
+Use the <TypeLink links={docs.links} type={docs.exports.toCalendarDateTime} /> function to convert a `ZonedDateTime` to a `CalendarDateTime`.
+
+```tsx
+import {toCalendarDateTime} from '@internationalized/date';
+
+let date = parseZonedDateTime('2022-02-03T09:45[America/Los_Angeles]');
+toCalendarDateTime(date); // 2022-02-03T09:45:00
+```
+
+### Between time zones
+
+A `ZonedDateTime` can be converted to a different time zone using the <TypeLink links={docs.links} type={docs.exports.toTimeZone} /> function. The <TypeLink links={docs.links} type={docs.exports.toLocalTimeZone} /> function can also be used to convert to the current user's local time zone.
+
+```tsx
+import {toTimeZone, toLocalTimeZone} from '@internationalized/date';
+
+let date = parseZonedDateTime('2022-02-03T09:45[America/Los_Angeles]');
+toTimeZone(date, 'America/Chicago'); // 2022-02-03T11:45[America/Chicago]
+toLocalTimeZone(date); // e.g. 2022-02-03T12:45[America/New_York]
+```
+
+## Queries
+
+### Comparison
+
+`ZonedDateTime` objects can be compared either for full or partial equality, or in order to determine which date is before or after another.
+
+The `compare` method can be used to determine if a date is before or after another. It returns a number less than zero if the first date is before the second, zero if the values are equal, or a number greater than zero if the first date is after the second.
+
+```tsx
+let a = parseZonedDateTime('2022-02-03T09:45[America/Los_Angeles]');
+let b = parseZonedDateTime('2022-03-04T09:45[America/Los_Angeles]');
+
+a.compare(b) < 0; // true
+b.compare(a) > 0; // true
+```
+
+In addition, the following functions can be used to perform a partial comparison. These functions accept dates in different calendar systems, and the second date is converted to the calendar system of the first date before comparison.
+
+* <TypeLink links={docs.links} type={docs.exports.isSameYear} /> – <span>{docs.exports.isSameYear.description}</span>
+* <TypeLink links={docs.links} type={docs.exports.isSameMonth} /> – <span>{docs.exports.isSameMonth.description}</span>
+* <TypeLink links={docs.links} type={docs.exports.isSameDay} /> – <span>{docs.exports.isSameDay.description}</span>
+* <TypeLink links={docs.links} type={docs.exports.isToday} /> – <span>{docs.exports.isToday.description}</span>
+
+```tsx
+import {isSameMonth, IslamicUmalquraCalendar} from '@internationalized/date';
+
+isSameMonth(
+  parseZonedDateTime('2022-04-16T09:45[America/Los_Angeles]'),
+  parseZonedDateTime('2022-04-30T05:15[America/Los_Angeles]'),
+); // true
+
+isSameMonth(
+  parseZonedDateTime('2022-04-16T09:45[America/Los_Angeles]'),
+  parseZonedDateTime('2022-08-020T05:15[America/Los_Angeles]'),
+); // false
+```
+
+A similar set of functions is also available that does not convert between calendar systems and requires the calendars to be equal.
```

Comment:
In other places you have examples of `isEqualXXXX`.

Reply:
Yeah it's just harder to show because ZonedDateTime has so many arguments for the constructor.